lib/rdoc/generator/template/rails/index.rhtml

```diff
@@ -24,9 +25,8 @@
         </ul>
     </div>
 
-    <div id="bodyContent">
+    <main id="bodyContent">
```

Comment:
I'm super excited for this seeing the effect of changing this tag. I started tracking a few other sites, and it gave a pretty decent SEO boost.

Reply:
Landmarks are super important!